stack.yaml

```diff
@@ -28,8 +28,8 @@ packages:
     commit: 95fce1dbada62020a0b2d6aa2dd7e88eadd7214b
   extra-dep: true
 - location:
-    git: https://github.com/serokell/kademlia.git
-    commit: bbdca50c263c6dae251e67eb36a7d4e1ba7c1cb6
+    git: https://github.com/avieth/kademlia.git
+    commit: b86e3cb717c3398ddc1cc25cfb8f8ea3491e4026
```

Comment:
I hope so! I needed some changes to the kademlia library, but I don't have write access to `serokell/kademlia`. I'll make a pull request over there so we can go back to the serokell variant.